test/app-spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import test from 'blue-tape';
```

Comment:
I think it would be better to use Jest rather than blue-tape/faucet. Based on a few things:
- Currently testing for mobile and our react-native components, while not actually written or up to date, are set up. Having the tests using the same syntax across projects will be far easier for all (that said, what we decide on should go into the template as I have said in other PRs).
- Backed by facebook and is far more broadly used. Made with react and RN in mind. Jasmine style test syntax is good in my books.
- Jest has some sweet built in tools that may be good to utilise in the future, in particular live tests that run while editing files and snapshot testing.

If you have a strong case for this set up I want to hear it though ;)